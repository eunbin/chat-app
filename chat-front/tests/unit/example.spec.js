import { mount } from '@vue/test-utils'
import App from '@/App.vue'
import './test-setup'

describe('App.vue', () => {
  it('snapshot check', () => {
    const wrapper = mount(App, {
      stubs: ['router-link', 'router-view']
    })

    expect(wrapper.vm.$el).toMatchSnapshot()
  })

  it('application id check', () => {
    const wrapper = mount(App, {
      stubs: ['router-link', 'router-view']
    })

    expect(wrapper.find('div').attributes('id')).toEqual('chat-app')
  })
})
