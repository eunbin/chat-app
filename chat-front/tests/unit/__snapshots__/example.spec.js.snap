// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue snapshot check 1`] = `
<div
  class="application theme--light"
  data-app="true"
  id="chat-app"
>
  <div
    class="application--wrap"
  >
    <main
      class="v-content"
      style="padding: 0px 0px 0px 0px;"
    >
      <div
        class="v-content__wrap"
      >
        <router-view-stub />
      </div>
    </main>
  </div>
</div>
`;
